app/src/main/res/values/strings.xml

```diff
@@ -64,4 +64,19 @@
         <item>Other</item>
     </string-array>
 
+    <!-- TODO: Remove or change this placeholder text -->
```

Comment:
This seems like something we should file an issue for

Reply:
Done